apertium-en-pt.pt.dix

```diff
@@ -2,78 +2,78 @@
 <dictionary>
   <alphabet>ÀÁÂÃÄÇÈÉÊËÌÍÎÏÑÒÓÔÕÖÙÚÛÜàáâãäçèéêëìíîïñòóôõöùúûüABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz</alphabet>
   <sdefs>
-    <sdef n="acr"/>
-    <sdef n="predet"/>
-    <sdef n="detnt"/>
-    <sdef n="loc"/>
-    <sdef n="ant"/>
-    <sdef n="al"/>
-    <sdef n="atn"/>
-    <sdef n="enc"/>
-    <sdef n="pro"/>
-    <sdef n="ND"/>
-    <sdef n="GD"/>
-    <sdef n="tn"/>
-    <sdef n="n"/>
-    <sdef n="np"/>
-    <sdef n="adj"/>
-    <sdef n="f"/>
-    <sdef n="m"/>
-    <sdef n="mf"/>
-    <sdef n="sg"/>
-    <sdef n="pl"/>
-    <sdef n="adv"/>
-    <sdef n="preadv"/>
-    <sdef n="pr"/>
-    <sdef n="prn"/>
-    <sdef n="rel"/>
-    <sdef n="nn"/>
-    <sdef n="an"/>
-    <sdef n="aa"/>
-    <sdef n="ind"/>
-    <sdef n="itg"/>
-    <sdef n="det"/>
-    <sdef n="dem"/>
-    <sdef n="def"/>
-    <sdef n="cnjcoo"/>
-    <sdef n="cnjsub"/>
-    <sdef n="cnjadv"/>
-    <sdef n="nt"/>
-    <sdef n="vbser"/>
-    <sdef n="vbhaver"/>
-    <sdef n="vblex"/>
-    <sdef n="vbmod"/>
-    <sdef n="inf"/>
-    <sdef n="infps"/>
-    <sdef n="ger"/>
+    <sdef n="acr"        c="Acronym"/>
```

Comment:
could you have the spacing either be a single space or have all the definitions aligned?